ui/.env

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_BACKEND_URL=https://thesophiaxu-oughtinc-ice-6vxpvq6prr52x4wq-8935.githubpreview.dev
```

Comment:
In `.gitignore`, can you change `/.env` to `**/.env`, and then revert this file?

Reply:
Done